src/components/Button.js

```diff
@@ -0,0 +1,12 @@
+import React, {Fragment} from "react";
+
+function Button({name, message, disabled}) {
+    return (<Fragment>
+            <button type="button" disabled={disabled} onClick={() => {
+                console.log({message});
+            }}>{name}</button>
+        </Fragment>
+    )
+}
+
```

Comment:
De button hebt je onnodig in een fragment gewrapt, omdat de button een enkel element is hoeft er geen fragment omheen